README.md

```diff
@@ -65,6 +65,25 @@ Or [download a ZIP of the files](https://github.com/digininja/DVWA/archive/maste
 
 ## Installation
 
+## Automated Installation 🛠️
+
+### Installation Steps
```

Comment:
Add a bit of a description what this does and what platform it is for otherwise someone will try to run these commands from a DOS prompt and then complain they don't work.

Reply:
Should I add more or its now ok?